testing/sdk_install.py

```diff
@@ -90,10 +90,6 @@ def install(
         insert_strict_options=True):
     start = time.time()
 
-    # If the package is already installed at this point, fail immediately.
-    if sdk_marathon.app_exists(service_name):
```

Comment:
Is this really what we want? We need to be careful making changes to `testing/` that are not ported to `dcos-commons` as these will be overwritten the next time we update the tooling.

Reply:
Why do we have this to begin with? Was it for CI/soak?

Without this, it is difficult to re-run the scale tests if there is a failure somewhere in the chain. But as @mpereira suggested, it might be better to include a flag so that auto-fail is optional.

Reply:
I think in the absence of a thorough verification that a similarly-named service is being installed has the same configuration, failing immediately might avoid some possible confusion.

Reply:
cc @mpereira. Could we revert this change? This would cause a lot of silent failures in our `dcos-commons` CI?

Reply:
https://github.com/mesosphere/spark-build/pull/348/commits/23c19ef3f3a0b01482cd74396d616d5bdd990239